packages/hot-updater/src/commands/deploy.ts

```diff
@@ -89,6 +90,19 @@ export const deploy = async (options: DeployOptions) => {
         })
       : null);
 
+  const outputPath =
+    options.bundleOutputPath ??
+    (options.interactive
+      ? (
+          await p.text({
+            defaultValue: cwd,
+            initialValue: cwd,
+            message: "Bundle.zip output path (e.g., ./dist/bundle/ios)",
+            placeholder: cwd,
+          })
+        ).toString()
+      : cwd);
```

Comment:
```suggestion
  const outputPath =
    options.bundleOutputPath ?? cwd;
```

It doesn’t seem important enough to require keyboard input, so I’ll exclude it from the interaction mode.

Reply:
I agree. I'll drop this change 👍